Source/Renderer/Context.js

```diff
@@ -508,6 +509,18 @@ define([
         },
 
         /**
+         * <code>true</code> if the WebGL context supports stencil buffers.
+         * Stencil buffers are not supported by all systems.
+         * @memberof Context.prototype
+         * @type {Boolean}
+         */
+        stencilBuffer : {
+            get : function() {
+                return this._stencilBits >= 8;
```

Comment:
I get that it is fine in practice (we think), but are you sure that `> 0` isn't more correct according to the spec?